tests/unit/core/test_installAddon.js

```diff
@@ -992,29 +1001,37 @@ describe(__filename, () => {
 
     describe('install', () => {
       const installURL = 'https://mysite.com/download.xpi';
+      let store;
+
+      beforeEach(() => {
+        store = createStore().store;
+      });
 
       it('calls addonManager.install()', () => {
         const hash = 'some-sha-hash';
-        const addon = createInternalAddon(
-          createFakeAddon({
+
+        _loadVersions({
+          store,
+          versionProps: {
             files: [
               {
                 platform: OS_ALL,
                 url: installURL,
                 hash,
               },
             ],
-          }),
-        );
+          },
+        });
+
         const fakeAddonManager = getFakeAddonManagerWrapper();
         const { root } = renderWithInstallHelpers({
           _addonManager: fakeAddonManager,
-          addon,
           defaultInstallSource,
+          store,
         });
         const { install } = root.instance().props;
 
-        return install(addon).then(() => {
+        return install().then(() => {
```

Comment:
mmh, interesting.

Reply:
Yeah, I tried and tried to understand why we were passing `addon` into `install` at all, and could not. Then discovered we don't even need it.